src/NUnitFramework/framework/Internal/Execution/EventPump.cs

```diff
@@ -191,7 +191,7 @@ private void PumpThreadProc()
             }
             catch (Exception ex)
             {
-                log.Error( "Exception in pump thread", ex );
+                log.Error("Exception in pump thread {0}", ex);
```

Comment:
Any reason you are not using `ExceptionHelper` in this repo (similar to the changes in the console-repo)?

And should we also make line 188 consistent with this line and remove `\r\n` ?

Reply:
Done! 🙂